examples/asset/utilization/AssetUtilization.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NationalInstruments.SystemLink.Clients.AssetManagement;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Asset.Utilization
+{
+    /// <summary>
+    /// Example for the SystemLink Asset Management Client API that
+    /// tracks utilization for assets present in the system.
+    /// 
+    /// Note: As a prerequisite for this example, ensure SystemLink Client is installed.
+    /// </summary>
+    class AssetUtilization
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * The Asset Utilization Store enables asset utilization tracking. 
+             * It is created using a factory.
+             */
+            var utilizationStoreFactory = new AssetUtilizationStoreFactory();
+            var assetUtilizationStore = utilizationStoreFactory.CreateAssetUtilizationStore();
```

Comment:
Should the store also get disposed? I don't remember what all is IDisposable.

Reply:
Yes. I am wrapping it in a using statement